src/Umbraco.Core/Models/ContentEditing/ContentItemDisplay.cs

```diff
@@ -135,6 +136,7 @@ public ContentItemDisplay()
     public DateTime UpdateDate { get; set; }
 
     [DataMember(Name = "template")]
+    [JsonPropertyName("template")]
```

Comment:
We can remove the `[DataMember(Name = "template")]` once Newtonsoft is entirely gone, but the other one is still required because the property is named`TemplateAlias` and not `Template`, but then again, the entire class can be removed once the old project is entirely gone 😄